github/github.go

```diff
@@ -986,6 +996,59 @@ func (g *GitHub) UpdateReleasePage(
 	return release, nil
 }
 
+// toRepositoryRelease builds a repository release from the set of options
+func (urpo *UpdateReleasePageOptions) toRepositoryRelease() *github.RepositoryRelease {
+	request := &github.RepositoryRelease{}
+
+	if urpo.Name != "" {
+		request.Name = &urpo.Name
+	}
+	if urpo.Body != "" {
+		request.Body = &urpo.Body
+	}
+
+	request.Draft = &urpo.Draft
+	request.Prerelease = &urpo.Prerelease
+
+	if urpo.Latest {
+		request.MakeLatest = ptr.To("true")
+	} else {
+		request.MakeLatest = ptr.To("false")
+	}
+
+	return request
+}
+
+// UpdateReleasePageWithOptions updates release pages (same as UpdateReleasePage),
+// but does so by taking a UpdateReleasePageOptions parameter. It will _not_ set
+// a release as latest unless the corresponding option is set.
+func (g *GitHub) UpdateReleasePageWithOptions(owner, repo string,
```

Comment:
One more thing I'm wondering about is if we should change `UpdateReleasePage` to use this function under the hood.

Reply:
I added this now, let me know if we want to undo it.